### PR TITLE
added basic test skeleton for #19

### LINKED ISSUE #29: Add continuous integration?
Travis, Circle? CI is fun.

[after #19]

### PR DESCRIPTION
**Pull request by audy**
Also closes #29.

### CODE REVIEW

**Comment by audy** [comment]
@acabunoc mind if I merge?

**Comment by acabunoc** [comment]
I'll review in a sec

File: package.json [hunk 1]
```diff
@@ -31,8 +33,10 @@
     "npm-run-all": "^1.2.4",
     "react": "^0.13.2",
     "react-router": "^0.13.3",
-    "webpack": "^1.8.11",
-    "dotenv": "~1.1.0"
+    "webpack": "^1.8.11"
+  },
+  "devDependencies": {
```
Comment (acabunoc) [inline package.json]: this is good - quick note for the future a few others can probably be moved under devDependencies (linters, etc)

**Comment by acabunoc** [comment]
r+ looks great!

good job

**Pull request merged**